project/currency_comparison/__tests__/currency_comparison.test.js

```diff
@@ -0,0 +1,50 @@
+const CurrencyComparison = require('../currency_comparison')
+const axios = require('axios')
+const { test } = require('@jest/globals')
+jest.mock('axios')
+
+const testSalary = new CurrencyComparison(50000)
+
+describe("Outputs the number of work hours needed to obtain an inputted salary in USD, earning foreign currency", () => {
```

Comment:
Since we don't teach the `describe()` function and since there seems to only be one set of tests, this should be removed